oidc-service-impl/src/test/java/io/mosip/esignet/services/AuthorizationHelperServiceTest.java

```diff
@@ -275,6 +287,19 @@ public void delegateAuthenticateRequest_withInvalidResult_thenFail() throws KycA
         }
     }
 
+    @Test
+    public void delegateAuthenticateRequest_withInValidDetails_thenFail() throws KycAuthException {
+        String transactionId = "transaction-id";
+        String individualId = "individual-id";
+        List<AuthChallenge> challengeList = new ArrayList<>();
+        OIDCTransaction oidcTransaction = new OIDCTransaction();
+        oidcTransaction.setRelyingPartyId("rp-id");
+        oidcTransaction.setClientId("client-id");
+        oidcTransaction.setAuthTransactionId("auth-transaction-id");
+        oidcTransaction.setRequestedClaims(new Claims());
+        Mockito.when(authenticationWrapper.doKycAuth(Mockito.anyString(), Mockito.anyString(), anyBoolean(), any(KycAuthDto.class))).thenThrow(KycAuthException.class);
+        Assert.assertThrows(EsignetException.class,()->authorizationHelperService.delegateAuthenticateRequest(transactionId, individualId, challengeList, oidcTransaction));
```

Comment:
can we assert a particular error code? Test cases can be more meaningful if the exact negative test case name is used instead of just calling it "withInvalidDetails"